### ISSUE TITLE
A question about the results of inverse problem(PINN)

### ISSUE THREAD

**Issue by GitHubSPlus** [lululxvi/deepxde#989]
Dear @lululxvi and team,

     I recently used the pinn library to solve the odes parameter prediction problem. In order to verify the applicability of my equation, I assumed unknown parameters, and then used the numerical solution results as the input of pinn, as the training set and test set, and then used pinn for prediction, and got perfect results. The parameter all-round prediction pair, and the pinn prediction results were also consistent with the input. The prediction results are inversely inserted into the odes numerical solution results are also consistent, these three curves coincide perfectly. And that proves the applicability of my equation. Now I transfer pinn's input to the experimental results, a prediction using pinn, pinn prediction results can also be very good and the input data for training and test loss are small, but the predicted parameters into the odes, found the result is not with the input and pinn prediction results on, sometimes even can't in good trend. As for this problem, I don't know whether the result of my experiment does not fit my equation well or what is the problem. Do you have any good suggestions on the procedure of this problem? I wanted to start with the loss function, but I don't understand it yet. Do you have any good suggestions?

**Comment by lululxvi** [comment]
It is possible that your experimental data is not consistent with the ODE.

**Comment by GitHubSPlus** [comment]
> 

Thank you for answering my question. By the way, when using PINN to predict parameters, can I set parameters in the initial conditions for prediction?I am now trying to set parameters in the initial conditions for PINN prediction, and I don't know  whether it is feasible.

**Comment by lululxvi** [comment]
> when using PINN to predict parameters, can I set parameters in the initial conditions for prediction?

This is not directly supported.
